src/server/session.ts

```diff
@@ -884,6 +915,11 @@ module ts.server {
                         response = this.getNavigationBarItems(navBarArgs.file);
                         break;
                     }
+                    case CommandNames.Occurrences: {
+                        var { line, offset, file: fileName } = <protocol.FileLocationRequestArgs>request.arguments;

```

Comment:
if you are using destructuring i would put it everywhere in this method for consistency.